assgnment2/util.dart

```diff
@@ -0,0 +1,20 @@
+import 'dart:io';
+
+class Util{
+
+static int validateInteger() {
```

Comment:
This doesn't look correct. The name of this function suggests that it would just validate any given integer but in reality it takes integer input until a valid one is provided.